docs/v1/P4Runtime-Spec.mdk

```diff
@@ -2185,6 +2185,114 @@ C++ API supports comparing messages while treating repeated fields as sets, so
 that different orderings of the same elements are considered equal. This method
 of comparing Protobuf messages may come at a cost in performance.
 
+### Data plane volatile objects
+
+An exception to read-write symmetry are objects whose contents or
```

Comment:
Nit: Should we briefly mention this exception and add a forward reference to this section at the beginning of the previous section, so people don't miss it?

Reply:
Done